services/ingestion/etl/budg/xls/src/lib/transform.js

```diff
@@ -202,7 +202,7 @@ const getProjectWebsite = record => record['Project Website'] || '';
 const getTypes = record =>
   (record['Activity type'] != null &&
     record['Activity type'] !== '' &&
-    record['Activity type'].split(',')) ||
+    record['Activity type'].trim().split(/\s*,\s*/)) ||
```

Comment:
Please split logic to make it easier to maintain. At the moment there are 2 checks, 2 operations and 1 default in 1 line which looks smart, but will be harder to debug if necessary.

Also, please keep original simple split and use map of trim afterwards istead, which is again for keeping things simple rather than regex which is an overkill here.

i.e. `.split(',').filter(a => a).map(a => a.trim())` is way simpler chain of operations to understand, no? (there's example in the [same file](https://github.com/ec-europa/eubfr-data-lake/blob/1a8cd11246b1b14bfed3d83da5b662d3b1af05db/services/ingestion/etl/budg/xls/src/lib/transform.js#L129) btw)